[typescript] - Material UI Icons JSX not a constructor error

I am experiencing an issue with the Material UI Icons (SVG) with typescript. 

- [x] I have searched the [issues](https://github.com/callemall/material-ui/issues) of this repository and believe that this is not a duplicate.

## Expected Behavior
I expect the following code to generate the menu icon inside of a div:
```
import * as React from 'react';
import MenuIcon from 'material-ui-icons/Menu';

const styles = (theme: any): any => ({ ... styles ... });

class ExampleComponent extends React.Component<any, any> {
  render() {
    return (
      <div>
        <MenuIcon />
      </div>
    );
}

export default withStyles(styles)(ExampleComponent);
```

## Current Behavior
This code instead generates the following error on `MenuIcon` (or any icon added):

```
[ts]
JSX element type 'Menu' is not a constructor function for JSX elements.
  Property 'render' is missing in type 'Menu'.
import MenuIcon
```

Also in the browser it displays the following:
```
Failed to compile
...\my-app\node_modules\material-ui-icons\Menu.d.ts
(2,35): error TS2507: Type 'ComponentType<SvgIconProps & StyledComponentProps<"root">>' is not a constructor function type.
This error occurred during the build time and cannot be dismissed.
```

When I remove the menu icon the app works as expected.

## Steps to Reproduce (for bugs)

1. Create an app with a typescript create react app wrapper:
https://github.com/wmonk/create-react-app-typescript
```
create-react-app testing-material-ui-icon-error-typescript --scripts-version=react-scripts-ts
cd testing-material-ui-icon-error-typescript/
```

2. Add material-ui and material-ui-icons:
`yarn add material-ui material-ui-icons` || `npm install material-ui material-ui-icons`

3. Start the server:
`yarn start` || `npm start`

4. Add the menu icon into the default /src/App.tsx:
```
import * as React from "react";
import "./App.css";
import MenuIcon from "material-ui-icons/Menu";
const logo = require("./logo.svg");

class App extends React.Component {
  render() {
    return (
      <div className="App">
        <div className="App-header">
          <MenuIcon />
          <img src={logo} className="App-logo" alt="logo" />
          <h2>Welcome to React</h2>
        </div>
        <p className="App-intro">
          To get started, edit <code>src/App.tsx</code> and save to reload.
        </p>
      </div>
    );
  }
}

export default App;
```

## Your Environment
Windows 10
Node 8.5.0
NPM 5.3.0

| Tech            | Version |
|--------------|---------|
| Material-UI  | 1.0.0-beta.13 |
| Material-UI-Icons | 1.0.0-beta.14 |
| React           |  16.0.0  |

Thank you for your time!

--- a/packages/material-ui-icons/create-typings.js
+++ b/packages/material-ui-icons/create-typings.js
@@ -21,14 +21,14 @@ function normalizeFileName(file) {
   return path.parse(file).name;
 }
 
-function createTypeDefinition(name) {
-  return `import SvgIcon from 'material-ui/SvgIcon';
-export default class ${name} extends SvgIcon {}`;
+function createTypeDefinition() {
+  // eslint-disable-next-line quotes
+  return `export { default } from 'material-ui/SvgIcon';`;
 }
 
 function createIconTyping(file) {
   const name = normalizeFileName(file);
-  return outputFile(path.resolve(TARGET_DIR, `${name}.d.ts`), createTypeDefinition(name), {
+  return outputFile(path.resolve(TARGET_DIR, `${name}.d.ts`), createTypeDefinition(), {
     encoding: 'utf8',
   });
 }

